apps/next/src/app/components/nav/nav.scss

```diff
@@ -60,5 +65,5 @@
 }
 
 .next-nav-link-active {
-  border-bottom: var(--color-primary-100) solid 2px;
+  border-bottom: var(--color-primary-100) solid $fluid-spacing--3x-small;
```

Comment:
Please do not use spacing tokens for border sizes. Eventually we will have tokens for border widths as well.

As a general rule, spacing tokens should only be used for things that create spacing, i.e.:
* margin
* padding
* top / left / right / bottom
* transforms
* grid gaps